capi/include/blazesym.h

```diff
@@ -301,7 +317,7 @@ typedef struct blaze_normalizer_opts {
    * may not be normalized successfully, as there is no reasonable way of
    * detecting staleness.
    */
-  bool cache_maps;
+  bool cache_vmas;
```

Comment:
We haven't yet released a stable version, so it really does not matter. For better or worse, we have already broken compat to `rc.0` since its release, so we might as well keep same order throughout, in my opinion.